gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-elimuModel = "model-2.0.100"
+elimuContentProvider = "1.4.22"
 elimuCommonutils = "1.0.13"
 elimuAnalytics = "3.5.11"
```

Comment:
@tuancoltech These two lines can be deleted, since they're included through the Content Provider dependency.

```diff
-elimuCommonutils = "1.0.13"
-elimuAnalytics = "3.5.11"
```